.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,23 @@
+name: Unit Tests
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: run unit tests
```

Comment:
please do not mix with check-tests, let's run them in parallel.
create a new file "tests.yml" here and run only unit tests